src/wp-admin/includes/update-core.php

```diff
@@ -18,6 +18,8 @@
  */
 global $_old_files;
 
+require_once ABSPATH . 'wp-includes/Requests/Exception.php';
```

Comment:
```suggestion
/**
 * Preload files from the current version into memory that may be
 * needed during the update process, and may not be available
 * after files from the updated version have been installed.
 *
 * Why? Requests 2.0.0 (introduced in WordPress 5.9.0) introduced
 * namespaces. If a class is needed after its file is upgraded but
 * the class is not already available in memory, a fatal error is thrown
 * because the fully-qualified name of the needed class is no longer
 * available in the filesystem.
 *
 * @since 5.9.0
 */
require_once ABSPATH . 'wp-includes/Requests/Exception.php';

global $_old_files;
```